Google.PowerShell/ReleaseFiles/GoogleCloudPowerShell.psd1

```diff
@@ -7,16 +7,16 @@
 #
 # Generated by: Google Inc
 #
-# Generated on: 12/17/2015
+# Generated on: 11/7/2016
 #
 
 @{
 
 # Script module or binary module file associated with this manifest.
-# RootModule = ''
+RootModule = '..\PowerShell\GoogleCloud\Google.PowerShell.dll'
 
 # Version number of this module.
-ModuleVersion = '0.1.0'
+ModuleVersion = '0.1.10.0'
```

Comment:
Should there be an extra .0 added? or should this be "0.1.10"? or should it be "0.1.01"?  The latter seems right to me